Google.Authenticator.Tests/AuthCodeTest.cs

```diff
@@ -21,5 +22,25 @@ public void BasicAuthCodeTest()
 
             actual.ShouldBe(expected);   
         }
+
+        [Fact]
```

Comment:
Thank you for writing a test :)
Two suggestions;
1. Don't use a dynamically generated time; hardcode a specific point in time for the test.
2. Could you add a test that (using a hardcoded point in time) shows that you get the same result from both the old and the new version of the methods? So you should show that `GeneratePINAtInterval` in both versions return the same PIN and that both versions of `ValidateTwoFactorPIN` return true.